settings.py

```diff
@@ -1404,7 +1396,7 @@ def _pkce_required(client_id):
             'propagate': False,
         },
         'smsbillables': {
-            'handlers': ['file', 'console', 'mail_admins'],
+            'handlers': ['file', 'console'],
```

Comment:
I look at this and think that mail_admins handler is still used. You're probably right, but can you explain that to me? It looks like right now if the smsbillables logger logs an ERROR or higher, that would trigger an email to admins?

Reply:
I have emails in my inbox that look like they come from https://github.com/dimagi/commcare-hq/blob/2fe91abdd26b1242049cd9bee491c1ce9950a24d/corehq/apps/smsbillables/utils.py#L17-L19

Reply:
Oh, interesting. Thanks for taking a deeper look, I also see them in my inbox too. Incidentally none of those emails are using the pygments (syntax highlighting) functionality, so maybe I can restore the handler but remove the code embedding/highlighting features?

Presumably those errors are also being log-shipped from the file logger to wherever the land for long-term storage and querying.

Next question: is anyone looking at those emails? Do we still need them? If not, I'd prefer to stick with the PR as is.